src/components/WebGLView/MouseController.ts

```diff
@@ -31,6 +31,8 @@ export class MouseController {
 
   onContext: (event: MouseEvent, button: number) => void;
 
+  onMouseDown: (event: MouseEvent) => void;
+
```

Comment:
are you sure this is required? most operations are either drag or click which are both already defined

Reply:
there even is a constant defined already^^

Reply:
no you are right. this is an artifact from when I used a different approach.